get_location.py

```diff
@@ -0,0 +1,53 @@
+"""Grab location from API based on users input. Currently set up to search for city names based on input only in the United States."""
+""" Currently when looking up by state only returns the state instead of all cities in the state"""
+import requests
+
```

Comment:
add `from classes import Location`

looks like classes file wasn't updated before you started the file. Will merge. Should pull latest updates, then branch and fix.